studio/deskStructure.js

```diff
@@ -27,10 +23,32 @@ export default () =>
         )
         .icon(MdHome),
       S.listItem()
-        .title('Pages')
-        .schemaType('page')
-        .child(S.documentTypeList('page').title('Page'))
-        .icon(MdLayers),
+        .title('Services Page')
+        .schemaType('servicesPage')
+        .child(
+          S.editor()
+          .id('servicesPage')
+          .schemaType('servicesPage')
+          .documentId('servicesPage'))
+        .icon(MdAssignment),
+      S.listItem()
+        .title('About Page')
+        .schemaType('aboutCompanyPage')
+        .child(
+          S.editor()
+          .id('aboutCompanyPage')
+          .schemaType('aboutCompanyPage')
+          .documentId('aboutCompanyPage'))
+        .icon(MdErrorOutline),
```

Comment:
A golden rule is – «Don't model your content for presentation». Read more about it here: https://www.sanity.io/docs/donts-and-dos